pkgs/development/libraries/glib/setup-hook.sh

```diff
@@ -1,9 +1,11 @@
 
 make_glib_find_gsettings_schemas() {
-    # For packages that need gschemas of other packages (e.g. empathy)
-    if [ -d "$1"/share/gsettings-schemas/*/glib-2.0/schemas ]; then
-        addToSearchPath GSETTINGS_SCHEMAS_PATH "$1/share/gsettings-schemas/"*
+  # For packages that need gschemas of other packages (e.g. empathy)
```

Comment:
Are not we supposed to use 4 space indent in shell scripts?

Reply:
Are we?

Reply:
According to [this](https://nixos.org/nixpkgs/manual/#sec-syntax), yes.

Reply:
Oh, thanks.